Application/EdFi.Ods.SandboxAdmin/Services/ClientCreator.cs

```diff
@@ -98,14 +107,22 @@ private ApiClient SetupDefaultSandboxClient(string sandboxName, SandboxOptions s
                 user.UserId,
                 defaultApplication.ApplicationId);
 
+            var connectionStringBuilder = _dbConnectionStringBuilderAdapterFactory.Get();
+
+            connectionStringBuilder.ConnectionString = _configConnectionStringsProvider.GetConnectionString("EdFi_Ods");
+            connectionStringBuilder.ApplicationName = "EdFi.Ods.WebApi"; 
+            connectionStringBuilder.DatabaseName = _databaseNameBuilder.SandboxNameForKey(apiClient.Key);
+
+            var connectionString = connectionStringBuilder.ConnectionString;
```

Comment:
@simpat-adam local variable is not needed here to store connection string, can be directly assigned in CreateOdsInstance